dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
 CWWKD1054.non.unique.result.useraction=Update the repository method to either use \
  a return type that supports multiple results or update the repository method \
  to return no more than a single result.
+
+CWWKD1055.unsupported.entity.prop=CWWKD1055E: The {0} property of the {1} entity \
+ has an unsupported type: {2}. Jakarta Data supports Java enumeration types, \
+ the {3} temporal types, and the following basic types: {4}. The built-in \
+ Jakarta Data provider also allows some Jakarta Persistence types, such as \
+ embeddables.
+CWWKD1055.unsupported.entity.prop.explanation=The entity property has a type \
+ that is not supported in Jakarta Data.
+CWWKD1055.unsupported.entity.prop.useraction=Update the entity property to use \
+ one of the supported types.
+
+CWWKD1056.unsupported.keyword=CWWKD1056E: The {0} method of the {1} repository \
+ interface has a name that includes a keyword, {2}, that is not a supported \
+ keyword of the Jakarta Data Query by Method Name pattern.
+CWWKD1056.unsupported.keyword.explanation=The keyword is not supported in the \
+ name of a repository method.
+CWWKD1056.unsupported.keyword.useraction=Rename the repository method to not use \
+ the unsupported keyword.
+
+CWWKD1057.no.args.constr.inacc=CWWKD1057E: The {0} return type of the {2} method \
+ of the {3} repository interface is not valid for a repository method because the \
```

Comment:
```suggestion
 in the {3} repository interface is not valid for a repository method because the \
```

Reply:
Using the word "in" here rather than "of" seems fine. However, it would be inconsistent with the 39 occurrences of the same phrase, "{0} method of the {1} repository", that appear within this same NLS messages file prior to this point, all of which passed reviews from ID team members.  It seems wrong to change only this one.  We should either leave this one as is or change them all.